### PR TITLE
add otel in focus for june

### PR DESCRIPTION
**Pull request by austinlparker**
**Preview**: https://deploy-preview-2963--opentelemetry.netlify.app/blog/2023/otel-in-focus-06/

### CODE REVIEW

File: layouts/partials/banner.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div class="o-banner">
 
-[The OpenTelemetry Demo Turns 1(.4)](/blog/2023/demo-birthday/)!
+[Catch up on the latest releases](/blog/2023/otel-in-focus-06/)!
```
Comment (chalin) [inline layouts/partials/banner.md:5]:
```suggestion
[Discover what's new](/blog/2023/otel-in-focus-06/) in our latest releases!
```

File: content/en/blog/2023/otel-in-focus-06.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+everyone who has contributed. Thank you!
+
+### From the blog...
+
+[End-User Q&A: OTel at Farfetch](/blog/2023/end-user-q-and-a-03/) is the latest
+in the End-User Q&A series, featuring Iris Dyrmishi of Farfetch.
+
+[Creating a Kubernetes Cluster with Runtime Observability](/blog/2023/k8s-runtime-observability/)
+discusses how to use recent tracing features in Kubernetes components to better
+understand your k8s clusters.
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-06.md:152]: Make these into a list?

File: content/en/blog/2023/otel-in-focus-06.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  System.Text.Encodings.Web.
+- The Http instrumentation has been updated to fix an issue with missing metric
+  data in network failures and to improve performance by avoiding boxing of
+  common status code values.
+
+## Project and Community Updates
+
+### OTLP 1.0 is here!
+
+OTLP 1.0 has (or will be soon) released! We'll publish more details about what
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-06.md:137]:
```suggestion
OTLP 1.0 is out (or will soon be)! We'll publish more details about what
```

File: content/en/blog/2023/otel-in-focus-06.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+- The Exemplars feature has been marked as internal and will be added back in
+  the 1.6.\* prerelease versions.
+- A new overload has been added for configuring MeterProviderBuilders while the
+  IServiceCollection is modifiable.
+- The Console and OpenTelemetryProtocol exporters no longer support exporting
+  Exemplars.
+- The ASP.NET Core instrumentation has been updated to fix an issue where
+  baggage was cleared when the Activity stopped and to add a direct reference to
+  System.Text.Encodings.Web.
+- The Http instrumentation has been updated to fix an issue with missing metric
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-06.md:129]:
```suggestion
- The HTTP instrumentation has been updated to fix an issue with missing metric
```

File: content/en/blog/2023/otel-in-focus-06.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+#### [.NET](/docs/instrumentation/net/)
+
+[Version 1.5.1](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.5.1)
+has been released, as well as 1.5.0. 1.5.1 is a fix for issues introduced in
+1.5.0, and 1.5.0 consisted of the following major changes:
+
+- The bug introduced in 1.5.0-rc.1 that caused the "Build" extension to return
+  null has been fixed.
+- The Exemplars feature has been marked as internal and will be added back in
+  the 1.6.\* prerelease versions.
```
Comment (chalin) [inline content/en/blog/2023/otel-in-focus-06.md:121]: What's the `\*` for? Were you planning to add a footnote?

Comment (austinlparker) [reply content/en/blog/2023/otel-in-focus-06.md:121]: Looks like some escaping gone wild. Will fix.

**Review by chalin — APPROVED** [review]

**Pull request merged**